app/views/allocations/_missing_information.html.erb

```diff
@@ -16,12 +14,12 @@
     </tr>
     </thead>
     <tbody class="govuk-table__body">
-    <% @prisoners.each_with_index  do |prisoner, i| %>
```

Comment:
Worth extracting to partial?

Reply:
This has since changed to @ offenders_awaiting_tiering
Needed merging with master.